Dockerfile

```diff
@@ -1,67 +1,72 @@
-#
 # CDS production docker build
-#
-FROM python:3.5
+
+FROM python:3.5-slim
```

Comment:
The container was using Python 3 before, so I was assuming we want to switch with Invenio 3
also to Python 3?